app/components/note-list/note-list-item.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+export default class NoteListItem extends Component{
+	render() {	
+		return (
+			<div> 
+				<li>
+					{this.props.note}
```

Comment:
The idea here is to have a fancy component (preferably a card) not just a list component.

Reply:
done, card created and I tried to make it more appealing